ft_crossfrequencyanalysis.m

```diff
@@ -257,12 +274,18 @@
 
 end % switch method for actual computation
 
-crossfreq.label      = cfg.channel;
 crossfreq.crsspctrm  = crsspctrm;
 crossfreq.dimord     = dimord;
 crossfreq.freqlow    = LF;
 crossfreq.freqhigh   = HF;
 
+if docrosschan
+    crossfreq.labelcmb = labelcmb;
```

Comment:
In this representation it is ambiguous as to which channel is the phase providing signal, and which one is the amplitude highfreq signal.

Reply:
I try following the existing logic of the code, i.e. `freqlow` -> `freqhigh`, so that the first item of the pairs (see my comment on `labelcmb` above) corresponds to `freqlow`, while the second item of the pairs corresponds to `freqhigh`.

(I assume `freqlow` corresponds to phase  and `freqhigh` corresponds to `amplitude`.)

Reply:
That's I think a valid assumption ( although I don't wrote the code myself). It's OK to follow the code's convention, but what I meant is that perhaps it is useful to make this implicit convention explicit to the user (either in the documentation, or in the labelname). E.g. rather than having labelcmb = {'chan01' 'chan02' ; ...}
have labelcmb = {'chan01phase' 'chan02amp'} or so.

Reply:
I have added an explicit description of the output. Can you, please, take a look? I have kept the original convention ("low" and "high").

Reply:
you mean in the documentation section? I checked, and rephrased it a bit.

Reply:
yes. thanks